Tests/MapKitTests/MKMapViewTests.swift

```diff
@@ -35,7 +35,41 @@ final class MKMapViewTests: XCTestCase {
         let annotationViewWithAnnotation = mapView.dequeueReusableAnnotationView(withClass: MKPinAnnotationView.self, for: annotation)
         XCTAssertNotNil(annotationViewWithAnnotation)
     }
-
+    
+    func testWithEmptyItemArray() {
+        let mapView = MKMapView()
+        let meter = 500.0
+        let edgePadding = EdgeInsets(top: 50, left: 50, bottom: 50, right: 50)
+        let previous = mapView.visibleMapRect
+        mapView.zoom(to: [], meter: meter, edgePadding: edgePadding, animated: true)
+        
+        XCTAssert(mapView.visibleMapRect.contains(previous))
+    }
+    
+    func testWithOneItemArray() {
+        let mapView = MKMapView()
+        let meter = 500.0
+        let oneItemArray = [CLLocationCoordinate2D(latitude: 36.9751, longitude: 38.4243)]
+        let edgePadding = EdgeInsets(top: 50, left: 50, bottom: 50, right: 50)
+        mapView.zoom(to: oneItemArray, meter: meter, edgePadding: edgePadding, animated: true)
+        
+        let firstPoint = MKMapPoint(oneItemArray.first!)
+        XCTAssert(mapView.visibleMapRect.contains(firstPoint))
+    }
+    
+    func testWithMultiItemArray() {
+        let mapView = MKMapView()
+        let meter = 500.0
+        let edgePadding = EdgeInsets(top: 50, left: 50, bottom: 50, right: 50)
+        let multiItemArray = [CLLocationCoordinate2D(latitude: 36.9751, longitude: 38.4243),
+                              CLLocationCoordinate2D(latitude: 37.06622, longitude: 37.38332),
+                              CLLocationCoordinate2D(latitude: 41.00527, longitude: 28.97696)]
+        mapView.zoom(to: multiItemArray, meter: meter, edgePadding: edgePadding, animated: true)
+        
+        multiItemArray.forEach { (location) in
```

Comment:
I'm not a fan of using `forEach` if you can use a regular `for-in` loop.
```suggestion
        for location in multiItemArray {
```

Reserve `forEach` for when you need to chain expressions, e.g.
```Swift
array
    .map { ... }
    .forEach { ... }
```

Reply:
@guykogus OK. I've completed the change 👍